### DISCUSSION TITLE
Doesnt work filter for doshboard

### DISCUSSION THREAD

**Discussion by siddizi** [chartbrew/chartbrew#257] [Q&A]
Hi! 
I have a trouble this dashbord filters. I cant apply filter for dashbord. My values ​​are different.
Could you have help me pls? Thanks!

![image](https://github.com/user-attachments/assets/6aff15ec-68d5-4c66-842e-c80003b762a7)

**Comment by razvanilin** [comment]
I will take a look at this one and see if I can find the issue. Are you self-hosting Chartbrew?

**Comment by siddizi** [comment]
Hi!
Yes. I use docker image, mysql and nginx. My version is 3.9.0

**Comment by razvanilin** [comment]
I just added a fix for this issue on the `master` branch here and updated the docker `latest` tag with the changes. Feel free to re-open this discussion if you're still seeing the issues after updating ✌️
